docs/docker/docker-troubleshooting.md

```diff
@@ -134,5 +134,14 @@ Start docker-sync so your docker-sync containers and volumes will be recreated:
 docker-sync start
 ```
 
+## Application is slow on Mac
+We focus on enhancing performance of application on all platforms.
+With Docker for Mac and Docker for Windows there are known performance issues because all project files need to be synchronized from host computer to application running in virtual machine.
+On Mac we partially solved this by implementing docker-sync. 
+Docker-sync has some limits and that is reason why we use Docker native volumes for syncing PostgreSQL and Elasticsearch data.
+You can increase performance by deleting this volumes in your `docker-compose.yml` file.
+You will loose persistence of your data so they will be lost after you remove your containers e.g. during `docker-compose down`.
+It is up to you to decide which is better for you, if it is slower application but with persistent data or faster but with lost persistence.
```

Comment:
This sentence is too "aggressive". I think the better form will be "In some cases, performance can be more important than the persistence of the data. In this case, the configuration can be changed and these special volumes can be removed as described."